### ISSUE TITLE
Space bar not working in main window (works in Options)

### ISSUE THREAD

**Issue by advice2020** [doublecmd/doublecmd#1020]
### Version

Version: 1.1.0 latest alpha version as of 6/13/2023 Appimage versions both gtk and qt.
Noticed that versions from end of May had the same issue.

### Description

Noticed that for some reason space bar is not working in any part of DC main windows.  Inline rename, address bar, quick search / filter fields, etc.

Space bar works fine in Options, Find Search dialog fields

I tried logging out of desktop environment and logging back in and it still does the same thing.

Not sure if anyone else is experiencing this?
I apologize if it is just me.

**Comment by advice2020** [comment]
Something strange I might not understand about how AppImages fully function, but I downloaded fresh instances of  both the gtk and qt and both produced this error.  I thought about it and wondered maybe there was a hotkey assigned to space bar so I looked and sure enough there was, but it was a fresh copy I did not set this HK up.  I changed it in the qt version and now all of a sudden that one works, the gtk one works (and I did not change the HK in this version).
I also downloaded a version 6/14/2023 and these don't have a hotkey setting either.
